criu/sigacts-dump.c

```diff
@@ -0,0 +1,356 @@
+#include <unistd.h>
+#include <inttypes.h>
+
+#include <sys/stat.h>
+#include <sys/wait.h>
+#include <sys/mman.h>
+
+#include "common/config.h"
+#include "common/compiler.h"
+#include "types.h"
+#include "protobuf.h"
+#include "images/sa.pb-c.h"
+#include "images/timer.pb-c.h"
+#include "images/vma.pb-c.h"
```

Comment:
Probably the old story about headers (https://github.com/checkpoint-restore/criu/pull/990#discussion_r407265715) can be continued here, why do you need vma.pb-c.h ?

Reply:
Thanks for reviewing. I will try and remove the unnecessary headers.

Reply:
@Snorch I have removed the unnecessary headers.

Reply:
You do all the same mistakes again and again https://github.com/checkpoint-restore/criu/pull/990#discussion_r401401455

All external calls should have their headers, all headers for which you don't have any external call should be removed.

Reply:
@Snorch Oh Sorry, I will try and correct it.
The sigpending.c test is failing as the value of numsig is coming out to be one less than sent_sigs. any idea why ?